### PR TITLE
feat: add allowExternalNameServices to KubernetesIngress provider

### LINKED ISSUE #521: Chart needs flag to enable external services on kubernetesingress provider
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik-helm-chart/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### What version of the Traefik's Helm Chart are you using?

traefik-10.6.1

### What version of Traefik are you using?

traefik:2.5.3

### What did you do?

Deployed an Ingress that consumes an externalName Service.

### What did you see instead?

```
time="2021-11-07T23:03:24Z" level=error msg="Cannot create service: externalName services not allowed: traefik/local-demo-http" providerName=kubernetes ingress=rancher-demo namespace=traefik serviceName=local-demo-http servicePort="&ServiceBackendPort{Name:,Number:80,}"
```

### What is your environment & configuration?

- K3s v1.21.5+k3s2

### Additional Information

The chart has a flag for `providers.kubernetescrd.allowExternalNameServices=true` but does not have a flag for `providers.kubernetesingress.allowExternalNameServices=true`. 

In order to consume an externalName Service from an Ingress, the chart must be deployed with `--set "additionalArguments={--providers.kubernetesingress.allowExternalNameServices=true}"`.

### PR DESCRIPTION
**Pull request by faust64**
### What does this PR do?

Adds a missing option (providers.kubernetesingress.allowExternalNameServices)

### Motivation

Fixes #521

### More

- [X] Yes, I updated the [chart version](https://github.com/traefik/traefik-helm-chart/blob/9b32ed1b414cc0be1ad46bcb335fcfc93ded06f3/traefik/Chart.yaml#L5)

### Additional Notes

N/A

### CODE REVIEW

**Comment by kevinpollet** [comment]
Hello @faust64 and thanks for your pull request,

I think the chart version should be `10.7.0` as this PR is an enhancement.

Could you also add some tests?

**Comment by gretel** [comment]
works for me:

```
time="2021-12-11T19:01:25Z" level=warning msg="ExternalName service loading is enabled, please ensure that this is expected (see AllowExternalNameServices option)" providerName=kubernetescrd
time="2021-12-11T19:01:25Z" level=warning msg="ExternalName service loading is enabled, please ensure that this is expected (see AllowExternalNameServices option)" providerName=kubernetes
```

File: traefik/tests/traefik-config_test.yaml [hunk 1]
```diff
@@ -124,6 +124,15 @@ tests:
       - contains:
           path: spec.template.spec.containers[0].args
           content: "--providers.kubernetescrd.allowExternalNameServices=true"
+  - it: should allow external name services when specified in configuration
```
Comment (kevinpollet) [inline traefik/tests/traefik-config_test.yaml:127]: Can you merge this test with the one above, instead of duplicating it?

**Review by kevinpollet — CHANGES_REQUESTED** [review]

**Comment by kevinpollet** [comment]
Superseded by #533

**Pull request closed without merging**